src/pages/Donate/DonateSupportUs.vue

```diff
@@ -15,11 +15,10 @@
 					:show-disclaimer="false"
 				/>
 			</div>
-			<div class="small-12 large-5 columns">
-				<h4>Loans vs. Donations</h4>
-				<p>Kiva does not take a cut of loans, optional donations fund our operations.</p>
+			<div class="small-12 large-5 columns donation-meter-holder">
+				<donate-support-us-right-rail :content="donationCalloutsCG" />
 			</div>
-			<div class="small-12 columns donation-faq">
+			<div class="small-12 columns donation-faq donation-faq-holder">
```

Comment:
This `donation-faq` class isn't used

Reply:
> The 'globe' image on contentful is 48x48, it should be uploaded at at least double that resolution to support the 2x needed for high density screens - As it is now it looks blurry

Totally agree! Where did you get the larger versions? They weren't in the Figma files I have for this page. Either way it can just be swapped in contentful when we get the better version.

Reply:
Exported at 2x from Figma!

Reply:
also removed unused class